### PR TITLE
Switch from pyvows to nosetest

### PR DESCRIPTION
**Pull request by kkopachev**
Figured we should switch to nosetests from pyvows.

* thumbor itself switched already
* I have doubts vows were actually testing anything
* I think that would help with Python3 migration

### CODE REVIEW

**Comment by Bladrak** [comment]
Wow great work, thanks a lot! Looks good so far, there's only one test failing in the CI, would you mind looking into it? https://circleci.com/gh/thumbor-community/aws/248?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

File: vows/result_storage_vows.py [hunk 1]
```diff
@@ -1,187 +0,0 @@
...
-            ctx = Context(config=config, server=get_server('ACME-SEC'))
-            ctx.request = Request
-            ctx.request.url = 'my-image.jpg'
-
-            storage = Storage(ctx)
-
-            storage.put(IMAGE_BYTES, callback=callback)
-
-        def should_be_in_catalog(self, topic):
-            expect(topic.args[0]).to_equal([sha1('my-image.jpg'.encode('utf8')).hexdigest(), 'my-image.jpg'].join('/'))
```
[the remark this answers is not in the document]
Comment (Bladrak) [reply vows/result_storage_vows.py]: Yeah there were a few issues indeed. Thanks a lot for switching the lib :)

**Comment by kkopachev** [comment]
@Bladrak test fixed, should be good to go. 
I've changed signatures (added `@return_future`) of `put_crypto` and `put_detector_data`, but I didn't find any calls to them, so I think it's safe anyway.

**Comment by Bladrak** [comment]
Thanks, I'm away from computers at the moment but I'll take a look when I can, in a few days.

**Comment by Bladrak** [comment]
@kkopachev Merged, however the build seems to be failing, would you mind checking it?

**Comment by kkopachev** [comment]
Looks like it fails to checkout from git. Maybe you rotated your ssh key?

**Comment by Bladrak** [comment]
Hmm no but looks like it's an issue with the docker image, checking it. I'll let you know

**Pull request merged**